test_tools/example_gen/example_gen.cc

```diff
@@ -152,6 +155,14 @@ int take_action(r::live_model& rl, const char *event_id, int action) {
         if( rl.report_action_taken(event_id, &status) != err::success )
           std::cout << status.get_error_msg() << std::endl;
       break;
+    case CCB_BASELINE_ACTION: {// "ccb",
+      r::multi_slot_response response;
+      std::vector<int> baselines { 1, 0 };
+      if(rl.request_multi_slot_decision(event_id, JSON_CCB_CONTEXT, 0, response, &baselines[0], 2, &status) != err::success)
```

Comment:
Why are you only passing in baseline[0]?  the length of the baseline_actions passed in should equal the number of slots.